relays/client-substrate/src/client.rs

```diff
@@ -200,9 +200,14 @@ impl<C: Chain> Client<C> {
 		let number: C::BlockNumber = Zero::zero();
 		let genesis_hash_client = client.clone();
 		let genesis_hash = tokio
-			.spawn(async move {
-				SubstrateChainClient::<C>::block_hash(&*genesis_hash_client, Some(number)).await
-			})
+			.spawn(
+				// We can't use the clippy suggestion because we have to borrow
+				// `genesis_hash_client` for &'static.
+				#[allow(clippy::redundant_async_block)]
```

Comment:
@serban300 https://gitlab.parity.io/parity/mirrors/parity-bridges-common/-/jobs/2584199

Reply:
Thanks ! Reverted this change.